### PR TITLE
SUS-2012: Introduce rate limit for WikisApiController

### PR DESCRIPTION
**Pull request by mszabo-wikia**
ticket: https://wikia-inc.atlassian.net/browse/SUS-2012

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![INFO][INFO] [WikisApiController.class.php#L17](https://github.com/Wikia/app/blob/ba615dd7571d4b0588e157bf020f101543d60790/includes/wikia/api/WikisApiController.class.php#L17): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [WikisApiController.class.php#L18](https://github.com/Wikia/app/blob/ba615dd7571d4b0588e157bf020f101543d60790/includes/wikia/api/WikisApiController.class.php#L18): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'

File: includes/wikia/api/WikisApiController.class.php [hunk 1]
```diff
@@ -256,6 +264,29 @@ public function getWikiData() {
...
+	/**
+	 * Get the value of an array request parameter passed as CSV without performing
+	 * a potentially expensive explode+count of all items.
+	 *
+	 * @param string $paramValue
+	 * @param int $maxCount
+	 * @return array
+	 */
+	protected function getArrayParameter( $paramValue, $maxCount ) {
+		$paramValue = strtok( $paramValue, ',' );
```
Comment (macbre) [inline includes/wikia/api/WikisApiController.class.php:276]: Nice use of `token()` :+1:

**Review by macbre — APPROVED** [review]

**Comment by macbre** [comment]
Travis says ``Warning: Task "karma:main" not found. Use --force to continue.``

**Comment by mszabo-wikia** [comment]
@macbre Travis failure is unrelated, #12875 fixes it :)

**Pull request merged**